tests/e2e/sentinel/transitions/mark_for_outbound.js

```diff
@@ -75,12 +75,12 @@ const wipeTasks = () => getTasks()
   });
 
 describe('mark_for_outbound', () => {
-  afterEach(done => Promise.all([utils.revertSettings(), wipeTasks()]).then(done));
-  afterAll(done => utils.revertDb().then(done));
+  afterEach(() => Promise.all([utils.revertSettings(true), wipeTasks()]));
+  afterAll(() => utils.revertDb());
 
   describe('when external server is up', () => {
-    beforeEach(done => startMockApp().then(done));
-    afterEach(done => { stopMockApp(); done(); });
+    beforeEach(() => startMockApp());
+    afterEach(() => void stopMockApp());
```

Comment:
Is `void` needed here given `stopMockApp` does not return anything?

Reply:
nope, it isn't. I think the initial idea was that we shouldn't wait for the mock server to close but then additional stuff was added to that function.